docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+Multi Datacenter
+================
+
+As of version 0.2.0 Cassandra on Mesos allows for providing configuration
+to allow multiple datacenters to be aware of one another.
+
+
+## How it works
+
+Cassandra on Mesos allows an operator to configure the framework to provide the Datacenter and rack level information
+to Cassandra upon starting each server.
+
+Cassandra on Mesos can be configured to query other instances of Cassandra on Mesos to provide seed sets across
```

Comment:
Change to: "Cassandra on Mesos can be configured to query other instances of itself to provide seed sets across multiple clusters so that each of the clusters can coordinate and replicate data."